src/index.ts

```diff
@@ -254,6 +254,20 @@ export class Amplitude {
     return AmplitudeReactNative.uploadEvents(this.instanceName);
   }
 
+  /**
+   * Sets the minimum cutoff time in millisseconds for sessions to be considered distinct.
+   *
```

Comment:
should we include in the docstring comment that the default is 5 mins?

source:  https://github.com/amplitude/Amplitude-iOS/blob/68eea8100a84d26c3d81997851965ec3efbe61fc/Sources/Amplitude/AMPConstants.m#L69

https://github.com/amplitude/Amplitude-Android/blob/fd928b4db3d7bfa326fb3d22f66a35ebde8928f6/src/main/java/com/amplitude/api/Constants.java#L30

Reply:
Will do so. Makes sense as part of the function documentation